deltachat-ffi/deltachat.h

```diff
@@ -2113,6 +2117,13 @@ uint32_t        dc_lookup_contact_id_by_addr (dc_context_t* context, const char*
  * To add a number of contacts, see dc_add_address_book() which is much faster for adding
  * a bunch of addresses.
  *
+ * This will always create or lookup an address-contact,
+ * i.e. a contact identified by an email address,
+ * with all messages sent to and from this contact being unencrypted.
+ * If the user just clicked on an email address,
+ * you should first check [`Self::lookup_contact_id_by_addr`]/`lookupContactIdByAddr.`,
+ * and only if there is no contact yet call this function here.
```

Comment:
```suggestion
 * This will always create or look up an address-contact,
 * i.e. a contact identified by an email address,
 * with all messages sent to and from this contact being unencrypted.
 * If the user just clicked on an email address,
 * you should first check [`dc_lookup_contact_id_by_addr`],
 * and only if there is no contact yet, call this function here.
```
But i don't understand, is there any case when the UI code don't want to call `dc_lookup_contact_id_by_addr()` first? If there isn't, this function should do all the job, not relaying it to UIs.